### PR TITLE
Added default Debian packages to MKE.

### PR DESCRIPTION
**Pull request by armandgrillet**
## Jira Ticket
https://jira.d2iq.com/browse/D2IQ-69240

## Description of changes being made
Added packages installed by default with the Docker image used by our MKE containers.

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [X] Create your PR against `staging`, not `master`. 
- [ ] Provide an estimated date for deploying the doc change. Note: Improvements or fixes can be merged ASAP. 
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

**Comment by armandgrillet** [comment]
Right now this is only against 2.4.10-1.15.10, I will change all our support versions' docs once I get this first one reviewed.

File: pages/mesosphere/dcos/services/kubernetes/2.4.10-1.15.10/supported-versions/index.md [hunk 1]
```diff
@@ -30,6 +30,198 @@ DC/OS Kubernetes bundles the following versions of the base technology:
 Packages are versioned with an `a.b.c-x.y.z` format, where `a.b.c` is the version of the DC/OS integration and `x.y.z` indicates the version of Kubernetes.
 For example, `1.0.0-1.9.3` indicates version `1.0.0` of the DC/OS integration and version `1.9.3` of Kubernetes.
 
+# Debian packages installed by default
+
+The containers providing the Mesosphere Kubernetes Engine are based on Debian Buster. The control-plane nodes have these Debian packages included with the base image:
```
Comment (nLight) [inline pages/mesosphere/dcos/services/kubernetes/2.4.10-1.15.10/supported-versions/index.md:35]: @armandgrillet do the worker nodes have the same set of packages?

Comment (armandgrillet) [reply pages/mesosphere/dcos/services/kubernetes/2.4.10-1.15.10/supported-versions/index.md:35]: Control plane, private, and public nodes use the `mesosphere/dcos-kubernetes-node` image.

**Comment by armandgrillet** [comment]
The PR has now been updated to target the three latest MKE releases 🚀.

**Pull request merged**